docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md

```diff
@@ -108,6 +108,14 @@ Clone Daedalus repository and go to the root directory:
     [nix-shell:~/cardano-sl]$ cd
     [nix-shell:~]$ git clone https://github.com/input-output-hk/daedalus.git
     [nix-shell:~]$ cd daedalus
+
+Please make sure you have appropriate version of `npm`. You can use [nvm](https://github.com/creationix/nvm) for it:
+
+    [nix-shell:~/daedalus]$ nvm install 6.7.0
+    [nix-shell:~/daedalus]$ nvm use 6.7.0
+
+Now run `npm`:
```

Comment:
better would be do just do `nix-shell` in this directory, that will always give the same Nix